content/devconf-workshop.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "DevConf.cz Mini Workshop"
+date: 2022-06-02
+draft: true
+---
+
+# DevConf.cz Mini Workshop: Packit and Fedora: downstream story continues
+
+## Competition
+
+### Rules:
+
+- package for a team
+- points for various usage of Packit
+- sources: [packit.dev/docs](https://packit.dev/docs) and present Packit team members
```

Comment:
I am thinking about what will be the most straightforward source for people that never used Packit, maybe linking also https://packit.dev/docs/configuration/#packit-as-a-service where the configuration of the jobs is documented could be helpful?

Reply:
I am really not sure. There are various starting pages we can use:
* https://packit.dev/docs/
* https://packit.dev/docs/configuration/#packit-as-a-service
* https://packit.dev/docs/packit-service/
* https://packit.dev/docs/guide/
* https://packit.dev/docs/configuration/

The subpages usually miss some information -- after some thinking, I am inclined to use just https://packit.dev/docs/ where there are descriptions for subpages.

Reply:
yeah I get it, I was just thinking that I would maybe start with `[Onboarding Guide](https://packit.dev/docs/guide/) - How to start using Packit.` section  (which doesn't include info relevant for the workshop competition) if I would see the `https://packit.dev/docs/` for the first time...but it's true that otherwise the info is spread across multiple subpages, so we can leave it is at is and maybe mention the relevant subpages when talking about instructions for the competition